mep/common/solr.py

```diff
@@ -8,4 +8,17 @@ class SolrSchema(schema.SolrSchema):
 
     nationality = schema.SolrStringField(multivalued=True)
 
-    # relying on dynamic fields for now
+    # relying on dynamic fields for everything else; see index data
+    # methods and solr queryset aliases for specifics
+
+    #: copy fields for facets and variant search options
+    copy_fields = {
+        # string version of sort name for sort/facet
+        'sort_name_t': 'sort_name_sort_s_en',
+        # unicode/case folding version of names for searching
+        'name_t': 'name_txt_gen',
+        'sort_name_t': 'name_txt_gen',
```

Comment:
codefactor has flagged these - is it legal to have duplicate keys in a dict? should we consider a different data structure (maybe map `sort_name_t` to a list)?

Reply:
Good catch! I just checked parasolr and we have a solution for this - don't repeat the key, provide a list of copy field destinations. I'll correct.